documentation/docs/10-adapters.md

```diff
@@ -56,6 +56,23 @@ You can also use `adapter-static` to generate single-page apps (SPAs) by specify
 
 > You must ensure [`trailingSlash`](configuration#trailingslash) is set appropriately for your environment. If your host does not render `/a.html` upon receiving a request for `/a` then you will need to set `trailingSlash: 'always'` to create `/a/index.html` instead.
 
+#### Static sites on Apache
+
+For a Single Page Application on an `Apache` Server, you need a `.htaccess` to route all of the requests into the entry file for the routing to work, unless it will throw a 404 error once you refresh or visit a page that isn't compiled.
```

Comment:
```suggestion
For a Single Page Application on an `Apache` Server, you need an `.htaccess` to route all of the requests into the entry file for the routing to work, unless it will throw a 404 error once you refresh or visit a page that isn't compiled.
```